packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -1349,6 +1372,13 @@ class DDSMastheadComposite extends HostListenerMixin(LitElement) {
       this._isMobileVersion = layoutBreakpoint.matches;
       this.requestUpdate();
     });
+
+    // Watch for changes to dds-masthead's immediate children.
+    this._heightMutationObserver.observe(this.mastheadRef, {
+      childList: true,
+      subtree: false,
+      attributes: false,
+    });
```

Comment:
I like that we're watching for changes to try to keep the heights in sync, but I wonder if a [resizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) would be the better choice here.

Reply:
Good call.